addons/warns.py

```diff
@@ -1,12 +1,15 @@
 import discord
 import json
+import time
 from discord.ext import commands
 
-class Warning(commands.Cog):
 
+class Warning(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
-
+        self.warn_db_storage = bot.warn_db_storage
```

Comment:
You.. I... why? delete line 10. Change the if statement to `if bot.warn_db_storage`. There's no need to set it as a variable. It can be accessed via self.bot.warn_db_storage later anyways.